styles.css

```diff
@@ -331,3 +340,85 @@ body {
   color: #5e6c84;
   border: 1px solid #5e6c84;
 }
+
+.contact-section {
+  background-image: url("./assets/contactformshapes-mobile.svg"), url("./assets/Contactme.png");
+  background-repeat: no-repeat;
+  background-size: 130% 100%;
+  background-blend-mode: lighten;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-section h3 {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-top: 90px;
+  margin-bottom: 0;
+}
+
+.contact-section form {
+  width: 90%;
+}
+
+.contact-section p {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  padding: 10px;
+  color: #fff;
+  margin-top: 10px;
+}
+
+.contact-section textarea {
+  width: 100%;
+  height: 10rem;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.input-field-contact {
+  width: 100%;
+  height: 2rem;
+  margin: 10px auto;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+}
```

Comment:
[OPTIONAL]
- It would be better to add some paddings to your form inputs to improve your form design.